openedxstats/apps/sites/views.py

```diff
@@ -454,3 +453,11 @@ def bulk_update(request):
         resp['updated_over_count'] = False
 
     return json_response(data=resp)
+
+def get_valid_courses():
```

Comment:
Let's give this a better name.  This function doesn't get courses.  It makes a query for getting courses, so `valid_course_query` sounds better to me.

Reply:
(and update the docstring)